### DISCUSSION TITLE
v1.4.4

### DISCUSSION THREAD

**Discussion by buresdv** [buresdv/Cork#361] [Announcements]
## What's Changed
* A few optimizations

**Full Changelog**: https://github.com/buresdv/Cork/compare/v1.4.3...v1.4.4

<hr /><em>This discussion was created from the release <a href='https://github.com/buresdv/Cork/releases/tag/v1.4.4'>v1.4.4</a>.</em>

**Comment by kernelpanic-root** [comment]
Hello dev,

sorry for this dumb question. Do i need to self compiled every update? Bceause i have mac air m1 8gb , xcode lags so much. After self compiled, i have to uninstall xcode.

**Reply by buresdv** [reply]
Hey,

Yes, you have to compile it yourself with every update.

The bought version gives you access to the automatically updating pre-compiled version.
